app/src/main/res/values/strings.xml

```diff
@@ -1042,8 +1027,6 @@
     <string name="share_copy_link_to_clipboard">Copy to clipboard</string>
     <!-- Toast shown after copying link to clipboard -->
     <string name="toast_copy_link_to_clipboard">Copied to clipboard</string>
-    <!-- An option from the three dot menu to into sync -->
-    <string name="sync_menu_sign_in" moz:removedIn="107" tools:ignore="UnusedResources">Sign in to sync</string>
```

Comment:
This seems wrong you can only remove everything <= 105 since release is 106. https://wiki.mozilla.org/Release_Management/Calendar

Reply:
Ah, my mistake.  I'll update to <=105.  Thanks